package-lock.json

```diff
@@ -1,16 +1,15 @@
 {
   "name": "enact",
-  "version": "5.0.0-alpha.5",
```

Comment:
i'm concerned about deleting package-lock and then running a npm install for this case.
When we add/remove dependencies or update a major version of a dependency, we should not delete package-lock. Deleting will cause installs of latest minor versions of other dependencies and we did not trace any changelog/breaking changes for those.
Please reinstate original package-lock, perform an npm i and then push the file.